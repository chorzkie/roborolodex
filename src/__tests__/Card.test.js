import { shallow, mount, render } from 'enzyme';
import Card from '../components/Card';
import React from 'react';
import toJson from 'enzyme-to-json';


it('expect to render Card component', () => {
    expect(toJson(shallow(<Card />))).toMatchSnapshot();
});
