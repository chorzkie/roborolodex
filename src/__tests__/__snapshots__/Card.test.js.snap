// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card component 1`] = `
<div
  className="tc bg-light-blue dib br3 pa3 ma2 grow bw2 shadow-5"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?100x100"
  />
  <div>
    <h2 />
    <p>
      Username: 
    </p>
    <p>
      ID number: 
    </p>
    <p />
  </div>
</div>
`;
